src/libs/Clipboard/index.website.js

```diff
@@ -0,0 +1,80 @@
+import CONST from '../../CONST';
+import * as Browser from '../Browser';
+import MiscClipboardFunctions from './miscClipboardFunctions';
+
+const canSetHtml = MiscClipboardFunctions.canSetHtml;
+const setString = MiscClipboardFunctions.setString;
+
+/**
+ * Deprecated method to write the content as HTML to clipboard.
+ * @param {String} html HTML representation
+ * @param {String} text Plain text representation
+ */
+const copyToClipboardDeprecated = (html, text) => {
```

Comment:
I agree with Tgolen's comment https://github.com/Expensify/App/issues/10262#issuecomment-1338151467 but I think we can just put this in `index.js` file and we don't have to create worry about the desktop. This way it will look better.

this function is very small and does not take up much space.